index.js

```diff
@@ -31,8 +31,12 @@ function init() {
   ui.setCustomDatasetCallback(csv => selectDataset(data.parseCsv(csv)))
   ui.setHightlightCallback(id => canvas.getGrid().onHighlightGeo(id))
   ui.setUnhighlightCallback(() => canvas.getGrid().resetHighlightedGeo())
-  ui.setResolutionChangedCallback(value => {
-    hexagonGrid.setTileEdge(value)
+  ui.setResolutionChangedCallback((metricPerTile, sumMetric) => {
```

Comment:
I agree, This code should be pushed into the logic layer, but it is hard to say where. A value coming from `Ui` must be compared with a value in `MapGraphic` in order to set a property in `HexagonGrid`. There's no perfect place for this now, but even stashing it in `Canvas` ("`updateGridResolution`" or something) would probably be better than here.